go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/jackc/pgx/v4 v4.16.1
 	github.com/karlseguin/ccache/v2 v2.0.8
 	github.com/oklog/ulid/v2 v2.1.0
+	github.com/pressly/goose/v3 v3.6.1
```

Comment:
Can we upgrade this to use the new changes that introduced the goose `NopLogger` so that we don't get log output from the goose migrations?

Reply:
Done. Thanks!